maintainers/maintainer-list.nix

```diff
@@ -2222,6 +2222,13 @@
     githubId = 68566724;
     name = "bootstrap-prime";
   };
+  boozedog = {
+    name = "David A. Buser (boozedog)";
```

Comment:
```suggestion
    name = "David A. Buser";
```

Reply:
Could you also please sort the attributes alphabetically?

Reply:
> Could you also please sort the attributes alphabetically?

Done.